src/Command/CreateAdminCommand.php

```diff
@@ -53,6 +59,17 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             ['ROLE_ADMIN']
         ));
 
+        $question = new ConfirmationQuestion('Allow for sending anonymous usage statistic? (y/n) ', true);
```

Comment:
We will need to add a link to the documentation here, where it says exactly what data is collected.